scripts/ci/pre_commit/pre_commit_inline_scripts_in_docker.py

```diff
@@ -45,16 +44,16 @@ def insert_content(file_path: Path, content: list[str], header: str, footer: str
     SCRIPTS_DOCKER_DIR = AIRFLOW_SOURCES_DIR / "scripts" / "docker"
 
     for file in [DOCKERFILE_FILE, DOCKERFILE_CI_FILE]:
-        for script in listdir(SCRIPTS_DOCKER_DIR):
-            script_content = (SCRIPTS_DOCKER_DIR / script).read_text().splitlines(keepends=True)
+        for script in SCRIPTS_DOCKER_DIR.listdir():
```

Comment:
Use `scandir` instead

Reply:
Should by `iterdir()` actually.

Reply:
Oh right, I was confusing this with `os.scandir`.